src/components/dashboard/main-content/course-enrollments/course-cards/CompletedCourseCard.jsx

```diff
@@ -50,30 +50,30 @@ const CompletedCourseCard = (props) => {
       courseId: response.courseRunId,
       markedDone: response.markedDone,
     });
-    modifyIsUnarchiveCourseStatus({
+    modifyIsMoveToInProgressCourseStatus({
       isSuccess: true,
     });
   };
 
   const getDropdownMenuItems = () => {
-    // Only courses that are manually archived (markedDone) should show a unarchive option.
-    // if course is ended or completed, you cannot unarchive it
+    // Only courses that are manually saved for later (markedDone) should show an option to move back to in progress.
+    // if course is ended or completed, you cannot move it back to in progress
     // TODO: we also need to add || courseRunStatus === 'completed' once api returns correct status
-    //   right now it always returns completed upon using Archive
+    //   right now it always returns completed upon using Save course for later
     if (!markedDone || isCourseEnded(endDate)) { return []; }
     return ([
       {
-        key: 'unarchive-course',
+        key: 'move-course-to-in-progress',
         type: 'button',
         onClick: () => {
           setIsModalOpen(true);
-          sendTrackEvent('edx.learner_portal.course.unarchive.modal.opened', {
+          sendTrackEvent('edx.learner_portal.course.move_to_in_progress.modal.opened', {
             course_run_id: courseRunId,
           });
         },
         children: (
           <div role="menuitem">
-            Unarchive course
+            Continue Learning
```

Comment:
I believe the text we wanted to undo 'Save for later' was 'Move to In Progress', can we make that change to the text for the dropdown button? I think the other 'continue learning' button the main card area is intended to lead to LMS, while this button 'Move To In Progress' is just to revert back to the in progress section...